lib/hubspot/connection.rb

```diff
@@ -48,6 +48,25 @@ def put_json(path, options)
         end
       end
 
+      def patch_json(path, options)
```

Comment:
find out if the API you're using also works with `put`
if it doesn't, DRY with `put_json`, maybe using a private method with a `verb` params that will be used in a `public_send`

Reply:
must use patch, I grouped post, put and patch